### PR TITLE
Room 1 Updates

### PR DESCRIPTION
**Pull request by vivianliu0**
Loom Video;
https://www.loom.com/share/e67980fc756b4ef59ba39f16230146e6

Couple of notes:
- Teacher stance can't be modified in Ready Player Me, Jay will see if it can be modified through Blender
- Other two students have been left as is since the glb files from Ready Player Me are large

### CODE REVIEW

File: scripts/mockdata/assets_sample1.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   {
     "name": "Teacher",
     "obj_type": "object",
-    "s3_key": "assets/0881731b72a24889a72c03bd6e8f8770.gltf",
+    "s3_key": "assets/0881731b72a24889a72c03bd6e8f8770.glb",
```
Comment (ahmedhamodi) [inline scripts/mockdata/assets_sample1.json:23]: I really like the new teacher asset!

**Review by ahmedhamodi — COMMENTED** [review]
lgtm from a code perspective. Will leave to Aaron for design/content approval!

**Comment by vivianliu0** [comment]
Updated chemical formula to an image:
image
I noticed that when the image is very long and short, the text gets overlaid on top of the image

**Comment by ahmedhamodi** [comment]
> I noticed that when the image is very long and short, the text gets overlaid on top of the image

@drbalar can you look into this? You worked on the auto-scaling logic for the image and text. There might be an edge case here. (no rush, this can be during work session or after exams, but just wanted to make you aware)

**Review by aaronabraham311 — APPROVED** [review]

**Pull request merged**